templates/contrib/base.html

```diff
@@ -4,14 +4,10 @@
 
 {% block page_title %}Renseigner un établissement{% endblock%}
 
-{% block stylesheet_extras %}
-<link rel="stylesheet" href="{% static "vendor/selectWoo-1.0.8/css/select2.min.css" %}">
-<link rel="stylesheet" href="{% static "vendor/selectWoo-1.0.8/css/select2-bootstrap4.min.css" %}">
-{% endblock %}
-
 {% block javascript_extras %}
 <script src="{% static "vendor/selectWoo-1.0.8/js/select2.min.js" %}"></script>
 <script src="{% static "vendor/selectWoo-1.0.8/js/i18n/fr.js" %}"></script>
```

Comment:
These two are now imported in static/index.js so they shouldn't be included here anymore.